configs/minigpt4/minigpt-4_vicuna-7b_caption.py

```diff
@@ -57,10 +57,18 @@
     task='caption',
     prompt_template='###Human: {} ###Assistant: ',
     raw_prompts=[
-        '<Img><ImageHere></Img> Describe this image in detail.',
-        '<Img><ImageHere></Img> Take a look at this image and describe what you notice.',  # noqa
-        '<Img><ImageHere></Img> Please provide a detailed description of the picture.',  # noqa
-        '<Img><ImageHere></Img> Could you describe the contents of this image for me?',  # noqa
+        [
+            '<Img><ImageHere></Img> Describe this image in detail.',
+            '<Img><ImageHere></Img> Take a look at this image and describe what you notice.',  # noqa
+            '<Img><ImageHere></Img> Please provide a detailed description of the picture.',  # noqa
+            '<Img><ImageHere></Img> Could you describe the contents of this image for me?'  # noqa
+        ],
+        [
+            '<Img><ImageHere></Img> 详细描述这张图片。',
+            '<Img><ImageHere></Img> 浏览这张图片并描述你注意到什么。',
+            '<Img><ImageHere></Img> 请对这张图片进行详细的描述。',
+            '<Img><ImageHere></Img> 你能为我描述这张图片的内容吗？'
+        ]
```

Comment:
It's better to use a dict from language to prompts as the multi-language prompts. The list requires an order of languages implicitly, which may be confusing.

Reply:
Have changed the type of ``raw_prompt`` and ``prompt_template`` as ``dict``, which has two keys: ``en`` and ``zh``.